### DISCUSSION TITLE
Issue when using -d<VariableName>=<Name>

### DISCUSSION THREAD

**Discussion by HeyMarkSquires** [wixtoolset/issues#6520] [Questions] [answered]
I have a Wix variable here for setting which file I use for my license agreement:
`<WixVariabel Id="WixUILicenseRtf" Value="Licence.rtf"/>`
in the file Product.wxs.
I'm compiling through command line using 
`candle.exe Product.wxs`
and I'm trying to replace the value for 'WixUILicenseRtf" with a French localised version of the rtf file at link time using
`light.exe Product.wixobj -ext WixUIExtension -cultures:fr-fr -dWixUILicenseRtf=Licence-fr.rtf -loc Product_fr-FR.wxl -out French.msi`
and when I run that in the command line it gives me the error:
"light.exe : error LGHT0103 : The system cannot find the file '.rtf' with type 'Source'."

Does anyone have any suggestions?

**Comment by wyrdfish** [comment, marked as answer]
does it work with 
light.exe Product.wixobj -ext WixUIExtension -cultures:fr-fr -dWixUILicenseRtf="Licence-fr.rtf" -loc Product_fr-FR.wxl -out French.msi

**Reply by robmen** [reply]
Yes. The `WixVariable` element is used for *bind* time variables. Bind time occurs in `light.exe` so you set bind time variables from the `light.exe` command-line using the `-d` switch.
